.github/workflows/combine-xcframework.yml

```
@@ -71,6 +83,8 @@ jobs:
             -headers ./simulator/jdk/include \
             -output ./OpenJDK.xcframework
           zip -r OpenJDK.xcframework.zip OpenJDK.xcframework
+          zip -r java.base-device.zip java.base-device
+          zip -r java.base-simulator.zip java.base-simulator
 
       - name: Upload XCFramework
```

the name doesn't completely reflect what this step does. drop XCFramework or add java.base module
```suggestion
      - name: Upload XCFramework and java.base module
```